Explorer/Assets/Scripts/CrdtEcsBridge/JsModulesImplementation/Communications/SDKMessageBus/SDKMessageBusCommsAPIImplementation.cs

```diff
@@ -0,0 +1,50 @@
+using CRDT.Memory;
+using DCL.Multiplayer.Connections.Messaging;
+using Decentraland.Kernel.Comms.Rfc4;
+using SceneRunner.Scene;
+using SceneRuntime;
+using SceneRuntime.Apis.Modules.CommunicationsControllerApi.SDKMessageBus;
+using SceneRuntime.Apis.Modules.EngineApi.SDKObservableEvents.Events;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CrdtEcsBridge.JsModulesImplementation.Communications.SDKMessageBus
+{
+    public class SDKMessageBusCommsAPIImplementation : CommunicationsControllerAPIImplementation, ISDKMessageBusCommsControllerAPI
```

Comment:
Maybe it's better structure-wise to create a base class for both classes as they seem to be inherit common behaviour but `SDKMessageBusCommsAPIImplementation` does not really extend  `CommunicationsControllerAPIImplementation` but rather provides different behaviour for some of the functions. WDYT?

Reply:
Cool, sounds good, I did the minimum needed for that in this commit: https://github.com/decentraland/unity-explorer/pull/662/commits/84d3b36bcbfddc45541390b2d5258dae524be90c

Do you think that's enough or would you rather have the `CommunicationsControllerAPIImplementationBase` be an abstract class and replace the `ICommunicationsControllerAPI` everywhere it's being used to reference the abstract class instead? Although if we do that, probably more code changes will be needed in the future when we want to remove all this.